python/lib.test.w

```diff
@@ -0,0 +1,17 @@
+bring cloud;
+bring http;
+bring expect;
+bring "./lib.w" as python;
+
+let bucket = new cloud.Bucket();
+let func = new cloud.Function(new python.InflightFunction(
+  path: "./test-assets",
+  handler: "main.handler"
+).lift("bucket", bucket, allow: ["get", "put"]));
```

Comment:
```suggestion
let func = new cloud.Function(new python.InflightFunction(
  path: "./test-assets",
  handler: "main.handler",
  lift: {
    bucket: { obj: bucket, allow: ["get", "put"] }
  }
));
```

Maybe we can also support specifying lifts in props (in addition to the `lift()` method).